### ISSUE TITLE
Moodle Maintenance Cron Job 

### ISSUE THREAD

**Issue by allanhaggett** [warrenchristian1telus/moodle#17]
Moodle has an administrative maintenance script that is supposed to run via CRON regularly (every minute recommended). It is a vital part of how Moodle operates for synchronization and cleaning of various caches. 

Under traditional infrastructure, implementing a cron job is trivial; I have no idea if that is the case on OpenShift; I'm guessing it won't be difficult, but not making any assumptions.

**June 14** - Been difficult to get test case started due to issues with local environment; new image isn't quite ready and needs to be up and running in DEV.  Jared to check for support via FreshWorks. 

July 15 - Cron Job implemented; validation has been successful so far.

**Comment by warrenchristian1telus** [comment]
I've added a Liveness Probe to the test and production namespaces to both monitor liveness and trigger the cron every minute.

I've updated the team and waiting on additional testing/validation before marking this complete.

We may also want to update the call from HTTP (with password authentication) to a direct script call to minimize any security (most likely DOS-type) vulnerabilities.
